src/target/riscv/riscv-013_reg.c

```diff
@@ -0,0 +1,161 @@
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include "riscv-013_reg.h"
+
+#include "riscv_reg.h" // TODO: should be removed
```

Comment:
Could you please help me understand this TODO comment: what prevents you from removing this include right now?

Reply:
I've dropped the `TODO`.
I was thinking target-type-specific implementations (`riscv-01(1|3)_reg.c`) should only use private interface (`riscv_reg_impl.h`) and avoid using public one.
However, when I think of it now, it seems OK.